src/graphics/transformation.rs

```diff
@@ -63,6 +63,28 @@ impl Transformation {
     pub fn rotate(rotation: f32) -> Transformation {
         Transformation(Matrix3::new_rotation(rotation))
     }
+
+    /// Projects a point via this transformation.
+    ///
+    /// Can be used to convert world-coordinates to screen-coordinates
+    /// for a camera, for example.
+    pub fn project(&self, point: Point) -> Point {
+        let inverse_mat = self.0.try_inverse().unwrap();
```

Comment:
Given that we currently allow creating a `Transformation` from any kind of `Matrix3`, I think we should return an `Option<Point>` here so we can avoid panicking when the transformation is non-invertible.

An alternative is to only allow affine transformations, which should be more than enough and they are always invertible as long as its scaling transformation is non-zero. I think `nalgebra` has a type for this.

Reply:
Done!